demisto_sdk/commands/common/git_util.py

```diff
@@ -457,6 +459,13 @@ def renamed_files(
 
         return all_renamed_files
 
+    def get_all_changed_pack_ids(self, prev_ver: str) -> Set[str]:
+        return {
+            file.parts[1]
+            for file in self.get_all_changed_files(prev_ver)
+            if PACKS_FOLDER in file.parts
```

Comment:
```suggestion
            if file.parent.name == PACKS_FOLDER
```

Not just any folder.

I wonder what's the `parent.name` for unresloved files without a parent, though

Reply:
I'm not sure it will work.
What happens if an integration is changed? The parent of it is the integrations folder